gctscript/wrappers/gct/exchange/exchange_test.go

```diff
@@ -123,7 +123,7 @@ func TestExchange_AccountInformation(t *testing.T) {
 	if !configureExchangeKeys() {
```

Comment:
Update testname

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/98453e4de

Reply:
Since this is a slam dunk, I'm going to take the opportunity to rebase again and squash.